pkg/cli/cmd.go

```diff
@@ -95,6 +126,20 @@ func bindFlags(cmd *cobra.Command) {
 			val := viper.Get(f.Name)
 			cmd.Flags().Set(f.Name, fmt.Sprintf("%v", val))
 		}
+		// use CurrentProject as a backup for missing project flag
+		if !f.Changed && defaultForMissingFlagPossible(f.Name, "CurrentProject") {
```

Comment:
I wanted to try it, but didn't get the chance yet: what happens if you don't have a configuration file when you run `project set` ? Is one created automatically ?

Reply:
Good question, I did not test such particular case. I would assume it gets created, since it uses standard `viper` library calls and they should take care of it.